### ISSUE TITLE
just a suggestion add a furnace

### ISSUE THREAD

**Issue by frankhta** [Darkona/AdventureBackpack2#137]
is there any way to add a furnace like the workbench and that the Furnace takes out lava or charcoal or coal from the backpack so you can smelt all's or cook food on the go well you are adventuring around the world for 1.7.10

**Comment by connerfick** [comment]
I agree. Maybe as a separate tab in the UI with it's own inventory and the ability to use both liquid and solid fuels.
